addons/api/package.json

```diff
@@ -57,7 +58,7 @@
     "ember-page-title": "^7.0.0",
     "ember-qunit": "^8.1.0",
     "ember-resolver": "^10.0.0",
-    "ember-source": "~4.12.0",
+    "ember-source": "~5.12.0",
```

Comment:
We will need to clear peer dep. warnings (as a follow-up item) due to this update.